test_package/conanfile.py

```diff
@@ -1,8 +1,10 @@
 #!/usr/bin/env python
 # -*- coding: utf-8 -*-
-
-from conans import ConanFile, CMake, tools, RunEnvironment
+from __future__ import print_function
```

Comment:
I think it's better to use self.warn instead of print?

Reply:
Done!!!